src/package.json

```diff
@@ -2,45 +2,35 @@
   "name": "officehours",
   "version": "0.0.1",
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.28",
-    "@fortawesome/free-solid-svg-icons": "^5.13.0",
-    "@fortawesome/react-fontawesome": "^0.1.9",
-    "react": "^16.10.2",
-    "react-app-polyfill": "~0.2.2",
-    "react-bootstrap": "^1.4.0",
-    "react-bootstrap-dialog": "^0.13.0",
-    "react-dev-utils": "~8.0.0",
-    "react-dom": "^16.10.2",
-    "react-ga": "~2.7.0",
-    "react-phone-input-2": "^2.13.8",
-    "react-router-dom": "^5.0.0",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
+    "@fortawesome/free-solid-svg-icons": "^6.4.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.7.4",
+    "react-dom": "^18.2.0",
+    "react-ga": "~3.3.1",
+    "react-phone-input-2": "^2.15.1",
+    "react-router-dom": "^6.11.1",
     "reconnecting-websocket": "^4.4.0",
-    "yup": "^0.29.3"
+    "yup": "^1.1.1"
   },
   "scripts": {
     "prod": "webpack --config webpack.config.js --mode production",
-    "build": "webpack --config webpack.config.js --mode development --display-error-details",
-    "watch": "webpack --config webpack.config.js --watch --mode development --display-error-details",
```

Comment:
Why was `--display-error-details` removed? were they redundant as part of `webpack` upgrade?

Reply:
Yes, there was a deprecation warning, so I added the `stats.errorDetails` setting in `webpack.config.js`.